src/webots/gui/WbDragResizeEvent.cpp

```diff
@@ -90,15 +90,15 @@ void WbDragResizeHandleEvent::computeRatio(const QPoint &currentMousePosition) {
     mResizeRatio = 1.0;
     return;
   }
-  WbMatrix4 unscaledMatrix = mSelectedGeometry->matrix();
+  /*WbMatrix4 unscaledMatrix = mSelectedGeometry->matrix();
   WbVector3 absoluteScale = unscaledMatrix.scale();
   unscaledMatrix.scale(1.0f / absoluteScale.x(), 1.0f / absoluteScale.y(), 1.0f / absoluteScale.z());
 
   // compute and set position of detached handle on the axis
   WbVector4 handlePositionOnAxis(mAbsoluteScaleRatio * mViewDistanceUnscaling *
                                  mManipulator->relativeHandlePosition(mHandleNumber));
   handlePositionOnAxis[mCoordinate] = localMousePosition[mCoordinate] - mMouseOffset + mGeomCenterOffset;
-  handlePositionOnAxis = unscaledMatrix * handlePositionOnAxis;  // global position
+  handlePositionOnAxis = unscaledMatrix * handlePositionOnAxis;  // global position*/
```

Comment:
Same question here, does someone see any usefulness of this part of the code (same in src/webots/gui/WbDragScaleEvent.cpp )?

Reply:
When the handled were implemented this code was necessary to make the interaction work correctly so that the object, handle and mouse pointer were always moving together.

Now I don know if it is still needed. But it should be easy to check by testing the drag functionality.